html/janus.js

```diff
@@ -1864,6 +1864,29 @@ function Janus(gatewayCallbacks) {
 		callbacks = callbacks || {};
 		callbacks.success = (typeof callbacks.success == "function") ? callbacks.success : Janus.noop;
 		callbacks.error = (typeof callbacks.error == "function") ? callbacks.error : webrtcError;
+
+		function stopTracksOnError(stream, callbacks) {
```

Comment:
do we need to pass callbacks? Shall we just take `callbacks.error` from above directly?